### PR TITLE
Fixing small typos in documentation

### PR DESCRIPTION
**Pull request by akauppi**
These should be rather clear cases.

Unfortunately the PR is in multiple commits and I do not follow a commit message structure. Is that okay?

### CODE REVIEW

File: Documentation/dev-guide/api_reference_v3.md [hunk 1]
```diff
@@ -427,7 +427,7 @@ Empty field.
 | Field | Description | Type |
 | ----- | ----------- | ---- |
 | key | key is the first key to delete in the range. | bytes |
-| range_end | range_end is the key following the last key to delete for the range [key, range_end). If range_end is not given, the range is defined to contain only the key argument. If range_end is one bit larger than the given key, then the range is all the all keys with the prefix (the given key). If range_end is '\0', the range is all keys greater than or equal to the key argument. | bytes |
+| range_end | range_end is the key following the last key to delete for the range [key, range_end). If range_end is not given, the range is defined to contain only the key argument. If range_end is one bit larger than the given key, then the range is all the keys with the prefix (the given key). If range_end is '\0', the range is all keys greater than or equal to the key argument. | bytes |
```
Comment (heyitsanthony) [inline Documentation/dev-guide/api_reference_v3.md]: this is generated with `scripts/genproto.sh -g`; the correction should be in etcdserver/etcdserverpb/rpc.proto under DeleteRangeRequest (and should probably use wording similar to range_end's description for RangeRequest).

File: Documentation/dev-guide/api_reference_v3.md [hunk 2]
```diff
@@ -427,7 +427,7 @@ Empty field.
...
 | prev_kv | If prev_kv is set, etcd gets the previous key-value pairs before deleting it. The previous key-value pairs will be returned in the delte response. | bool |
```
Comment (heyitsanthony) [inline Documentation/dev-guide/api_reference_v3.md]: might as well s/delte/delete/ too

**Comment by fanminshi** [comment]
@akauppi you might want to merge all the commits before merge to master.

**Comment by akauppi** [comment]
@heyitsanthony @fanminshi I think I got it right, now.

**Comment by heyitsanthony** [comment]
lgtm. Thanks!

**Comment by fanminshi** [comment]
lgtm

**Pull request merged**